### PR TITLE
[example] Stepper UI words change

### PR DESCRIPTION
**Pull request by ibelem**
1. Update the stepper words simpler
2. Update the posenet model download issue since 404 from https://storage.googleapis.com/tfjs-models/weights/posenet/mobilenet_v1_101/manifest.json now

@Wenzhao-Xiang @BruceDai PTAL

### CODE REVIEW

File: examples/skeleton_detection/src/helperFunc.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ function getInput(inputElement) {
 
 function getURL(version) {
   let address;
-  const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
-  // const urlBase = '../skeleton_detection/model/';
+  // const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  const urlBase = '../skeleton_detection/model/';
```
Comment (pinzhenx) [inline examples/skeleton_detection/src/helperFunc.js]: it seems that we miss the model json & weights files

**Comment by BruceDai** [comment]
LGTM, thanks @ibelem

File: examples/skeleton_detection/src/helperFunc.js (hunk 1 above)
Comment (ibelem) [reply examples/skeleton_detection/src/helperFunc.js]: We are unable to commit/push these files to this repo which may have license issue. 

Thanks @pinzhenx for the review BTW, :)

Comment (Wenzhao-Xiang) [reply examples/skeleton_detection/src/helperFunc.js]: If catch the model from local, do we need to add a README.md about how to get posenet model?

Comment (ibelem) [reply examples/skeleton_detection/src/helperFunc.js]: I still have no good idea for users to get model if Google removed model json & weights files by json + bin and we don't have plan to update the new file support accordingly. Provide orginal model json & weights files in this repo may have license issue.

Comment (ibelem) [reply examples/skeleton_detection/src/helperFunc.js]: @Wenzhao-Xiang Will add link https://drive.google.com/open?id=1UcNbGZyIS8tt-OwhwiP6o1PxbK4V5j1i  in readme this noon.

Comment (ibelem) [reply examples/skeleton_detection/src/helperFunc.js]: @Wenzhao-Xiang The README have been updated via https://github.com/intel/webml-polyfill/pull/879/commits/2a1bf7b26829c9fdc381823eea6e2abbcb82772e , PTAL

Comment (Wenzhao-Xiang) [reply examples/skeleton_detection/src/helperFunc.js]: LGTM.

**Comment by Wenzhao-Xiang** [comment]
LGTM. thanks @ibelem

**Pull request merged**